samples/net/openthread/coap_client/README.rst

```diff
@@ -1,14 +1,15 @@
 .. _coap_client_sample:
 
-openthread: coap_client
-##################
+OpenThread: CoAP Client
+#######################
 
-The openthread coap_client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+The OpenThreag CoAP Client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
 
 Overview
 ********
 
-The client node can control the state of BSP_LED_3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
+The client node can control the state of LED 3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
+This sample uses Zephyr CoAP API for communication. For example usage of native OpenThread CoAp API refer to :ref:`coap_server_sample`
 
 
 Requirements
```

Comment:
Add in Requirements?
```
This sample requires a sample with a light resource programmed on a separate device.
```
You can define this line better by adding "Thread-compatible" or simply linking to the CoAP server if this sample cannot work with any other sample but this. I feel though that it is needed to give the information that this sample cannot work without another one.